resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -28,10 +28,19 @@ enum PointerSize {
     ExtraLarge
 }
 
+enum Status {
+        KeepCurrentValue
+	Enabled
+	Disabled
+}

```

Comment:
I made a mistake, you can remove this as it is better to just use a boolean.

Reply:
enum BinaryState {
    KeepCurrentValue = -1
	Enabled = 0
	Disabled = 1
}